### PR TITLE
Issue 143

### PR DESCRIPTION
**Pull request by isaahzorgh**

### CODE REVIEW

**Review by jklmnn — APPROVED** [review]

File: src/core/nRF52832/serial.adb [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+   ----------
+   -- Send --
+   ----------
+
+   procedure Send is
+   begin
+      TXD_PTR       := (PTR => String_Address (Buffer));
+      --   TXD_PTR := (PTR => String_Address (Str));
+      --   TXD_MAXCNT := (MAXCNT => Str'Length);
```
Comment (senier) [inline src/core/nRF52832/serial.adb]: Please remove commented code

File: src/core/nRF52832/serial.ads [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+   for Reg_TASK use record
+      TSK at 0 range 0 .. 0;
+   end record;
+
+   type BAUDRATE_Rate is (Baud1200, Baud2400, Baud4800, Baud9600,
+                          Baud14400, Baud19200, Baud28800, Baud38400,
+                          Baud57600, Baud76800, Baud115200, Baud230400,
+                          Baud250000, Baud460800, Baud921600, Baud1M) with
+     Size => 32;
```
Comment (senier) [inline src/core/nRF52832/serial.ads:135]: Defining this as an enum may be problematic when reading a `BAUDRATE_Rate` value from a register. Should this register have an invalid value (i.e. one not present in the representation clause) we may get an exception. Do we ever read a value of this type?

**Review by senier — CHANGES_REQUESTED** [review]
General remark: None of the files has a license header.

File: src/core/nRF52832/serial.ads (hunk 2 above)
Comment (jklmnn) [reply src/core/nRF52832/serial.ads:135]: This value should never be read.

Comment (senier) [reply src/core/nRF52832/serial.ads:135]: Then this should be safe.

**Review by jklmnn — APPROVED** [review]

**Review by senier — APPROVED** [review]

**Pull request merged**